forest/src/cli/chain_cmd.rs

```diff
@@ -32,9 +37,15 @@ pub enum ChainCommands {
         /// Include old messages
         #[structopt(short, long)]
         include_old_messages: bool,
-        /// Snapshot output path. Default to `forest_snapshot_{year}-{month}-{day}_height_{height}.car`
-        #[structopt(short, long)]
-        output_path: Option<String>,
+        /// Snapshot output path. Default to forest_snapshot_{chain}_{year}-{month}-{day}_height_{height}.car
```

Comment:
In the doc comment you can add `` around `forest_snapshot_{chain}_{year}-{month}-{day}_height_{height}.car` for better markdown display.

Reply:
Done